packages/graphql-transformers-e2e-tests/src/__tests__/IndexWithAuthV2WithFF.e2e.test.ts

```diff
@@ -271,7 +271,7 @@ test('listX with primaryKey', async () => {
 
   listResponse = await listFamilyMembersByParent(GRAPHQL_CLIENT_2, { parent: USERNAME1 });
   items = listResponse.data.byParent.items;
-  expect(items).toHaveLength(2);
+  expect(items).toHaveLength(1);
```

Comment:
This is the expected change due to the restricted records returned. It used to return both `sub::username` and `username` records. However, since the `child` is the sort key field of index `parent` and child is an owner field and missing in the input, it now resolves to the last item in the claim list, which cannot return both at the same time.